### DISCUSSION TITLE
Designing to address stress

### DISCUSSION THREAD

**Discussion by amycole501** [uswds/uswds#5869] [Accessibility]
April is [Stress Awareness Month](https://hr.nih.gov/working-nih/civil/national-stress-awareness-month), and it’s a good time to talk about the serious effect that stress has on how folks use our sites and services. Stress can take many forms. It can be an aspect of a health challenge or caretaking responsibilities. It can be related to employment, bereavement, or a natural disaster. It may not even have an identifiable cause. 

Stress is a very real part of the human experience, and the government services we build are often most critical in situations of stress. 

In the comments, let us know how you think about designing for stress. 

- What can teams do to help users who feel stressed? 
- What questions should we ask ourselves when we're building digital services that assist folks in times of stress? 
- Are there effective techniques for usability testing that can address stress while also respecting the vulnerability of those experiencing stress?

**Comment by sarah-sch** [comment]
User research with folks who've experienced stress is one option. Last spring, I saw a [really valuable talk on user research](https://www.confabevents.com/2022-speakers/helen-lawson) with participants who were terminally ill or who had lost a loved one. The research was conducted on behalf of a chain of funeral homes. Obviously it is important to be thoughtful about how to approach that kind of research, but the takeaway was that the research participants really appreciated being asked about their experience and having the opportunity to make things easier for the next person.

**Reply by sknep** [reply]
Similarly, the BEARS team built our prototype of the benefits tool while keeping empathy for these significant life events (Bereavement, in particular) front of mind. Other 10x projects in the last few years focused on [life events, disaster relief, and benefits for seniors](https://10x.gsa.gov/investments/01-investments-in-benefits-eligibility/), and here are some related resources:
- https://www.va.gov/survivors/faqs.asp 
- https://www.nia.nih.gov/health/grief-and-mourning/what-do-after-someone-dies
- https://labs.usa.gov/files/FFD_ResearchReport.pdf (pages 17 and 18)
- > “Life events such as the birth of a child, name changes, the passing of a relative, an accident, or job loss typically require public services across the three tiers of government. With a life events focus, these events and their ensuing paperwork can be curated into a single “service-on-a-page,” sparing citizens and businesses the complexity of having to navigate government agencies.” (Deloitte Insights - 2020)
- https://www.fema.gov/press-release/20240119/biden-harris-administration-reforms-disaster-assistance-program-help

**Reply by amycole501** [reply]
Such wonderful examples of using research and empathy to lessen stress on already stressful situations. Thank you for sharing!

**Comment by bonnieAcameron** [comment]
I was hit by an F4 tornado about a decade ago, and in the aftermath, I had to contact several utility companies to stop services to my unlivable apartment. I was so shaken that any executive task was extremely difficult--especially trying to read "walls of text" on a website. 

I keep that experience in mind now when I write or design for anything online:

- Keep the call to action clear, and design the page around that call to action
- Use scannable (or tab-able) content structure (Like "F structure") to promote easy reading and group similar concepts together

**Reply by amycole501** [reply]
What an impactful experience, Bonnie! Thank you for sharing. Going through a traumatic experience definitely impacts executive functioning. I like your suggestions on making content scannable and calls to action clear. Those techniques could be especially useful to help someone find relevant information when on a mobile device (often used during and after emergencies like that).

**Comment by justinmooreus0-prog** [comment]
This discussion about reducing stress while interacting with public services is really insightful.  

Many citizens also struggle with complex processes in government health schemes, like understanding eligibility and downloading their Ayushman Bharat (PMJAY) card. Clear, step-by-step guides can reduce stress and improve accessibility for first-time users.  

One such public-facing guide that explains eligibility, documentation, and the process clearly:
https://ayushmancard-download.online/

Sharing as an example of how thoughtful design and clear information can ease citizen experience.

**Comment by justinmooreus0-prog** [comment]
I really appreciate the focus on user experience here. Stress can make it so difficult for citizens to complete even simple government tasks, so clear design and step-by-step guidance is critical.
